Assets/Scripts/UI/PickupPup/DogProfile/DogProfileButtonController.cs

```diff
@@ -65,18 +65,20 @@ protected override void setReferences()
 
     #endregion
 
-	public void Init(DogProfile profile, DogDescriptor[] dogs)
+    public void Init(DogProfile profile, DogDescriptor[] dogs)
```

Comment:
It looks like this file is filled with empty changes like this. Safer to remove this from your PR.